routes/api/userRoutes.js

```diff
@@ -10,12 +12,12 @@ import {
   logoutUser,
 } from '../../controllers/userController.js';
 
-usersRouter.post('/register', registerUser);
+usersRouter.post('/register', validateRegistration, registerUser);
 
-usersRouter.post('/login', loginUser);
+usersRouter.post('/login', validateLogin, loginUser);
 
 usersRouter.post('/refresh', refreshTokens); //refresh token
 
-usersRouter.get('/profile', getUserProfile); // get user data (name mail)
+usersRouter.get('/profile', auth, getUserProfile); // get user data (name mail)
 
-usersRouter.get('/logout', logoutUser);
```

Comment:
Hej, tutaj ja zrobiłem błąd przy przygotowaniu szablonu w którym zorientowałem się dopiero przy testach na Postmanie - zamisat .get powinno być .post przy logout

Reply:
Z tym logoutem to w sumie można by było też GETem, tak jak było na LMSie w zadaniach domowych wymagane :). W sumie nie musimy nic wysłać, tylko gdy 'uderzymy' w ten endpoint to dzięki auth mamy dostęp do usera i czyścimy mu token. Tak w sumie robiłem w zadaniu domowym i mentor się nie przyczepił 😄

Reply:
Tak też się nad tym zastanawiałem i widzę że działa, ale doczytałem m.in tutaj:

https://www.baeldung.com/logout-get-vs-post

Lepiej zmienić, zrób commita napisz i pulluje

Reply:
Thx, done :)